### ISSUE TITLE
Package name changed

### ISSUE THREAD

**Issue by dbarboza** [pwsafe/pwsafe#90]
passwordsafe-ubuntu-0.98.1BETA.amd64.deb now provides package "pwsafe" whereas previous releases provided "passwordsafe". The Ubuntu package control file with 0.97 included `Conflicts: pwsafe` this is not a direct upgrade. Also, the package name in [(https://github.com/pwsafe/pwsafe/blob/master/install/deb/control-ubuntu.amd64)] does not match that in the deb in releases.

**Comment by vossenjp** [comment]
Also in https://sourceforge.net/p/passwordsafe/mailman/passwordsafe-linux/thread/20160403185316.GE15695%40blough.us/#msg34988857

This may be related, I'm not sure:
> On every Mint 17 (64-bit) and Ubuntu 14.04 LTS (32-bit) machine I've installed 0.98 on, the icon has disappeared from the GUI menu...  When I revert to 0.97 it works as expected.  I can consistently and reproducibly swap versions and see that behavior on at least 4 machines. So I've reverted all my machines to 0.97 for now.  Yet when I extract and compare the files in 97 and 98, "pwsafe.desktop" is identical.
>
> Side note: I'm using Ansible and (now) removing both "passwordsafe" and "pwsafe" before I copy the deb to tmp and "apt: deb=/tmp/passwordsafe-ubuntu-{{ passwordsafe_version }}.{{ deb_arch 
}}.deb state=present".

In addition to the GUI issue, something is very different with the packages themselves. I think the changes are in the deb control file(s) but I'm not finding the smoking gun on github. The package changed from "passwordsafe" to "pwsafe", the "Conflicts" line went away and the description is truncated, as follows.  Seems like something stuck on the buildhosts?  (This is also in the "[Passwordsafe-linux] 0.98.x beta vs Ubuntu 15.10" thread.)

And this is a long shot, but could the GUI menu problem there be that the "name" has changed yet the pwsafe.desktop file hasn't?

Steps to demonstrate name & package differences:
```
$ wget 
'https://github.com/pwsafe/pwsafe/releases/download/0.97BETA/passwordsafe-ubuntu-0.97BETA.amd64.deb' 
'https://github.com/pwsafe/pwsafe/releases/download/0.97BETA/passwordsafe-ubuntu-0.97BETA.i686.deb' 
'https://github.com/pwsafe/pwsafe/releases/download/0.98.1BETA/passwordsafe-ubuntu-0.98.1BETA.amd64.deb' 
'https://github.com/pwsafe/pwsafe/releases/download/0.98.1BETA/passwordsafe-ubuntu-0.98.1BETA.i686.deb'

$ for file in *; do echo $file; dpkg -I $file \
   | egrep '^ (Package|Conflicts):'; done
passwordsafe-ubuntu-0.97BETA.amd64.deb
  Package: passwordsafe
  Conflicts: pwsafe
passwordsafe-ubuntu-0.97BETA.i686.deb
  Package: passwordsafe
  Conflicts: pwsafe
passwordsafe-ubuntu-0.98.1BETA.amd64.deb
  Package: pwsafe
passwordsafe-ubuntu-0.98.1BETA.i686.deb
  Package: pwsafe

$ dpkg -I passwordsafe-ubuntu-0.97BETA.amd64.deb
  new debian package, version 2.0.
[...]
  Description: Portable version of popular password manager program
   pwsafe is the Linux version of the popular Windows PasswordSafe
   password manager.
   .
   Features:
    - Compatible with 1.x, 2.x and 3.x versions of the database format.
    - Uses wxWidgets for portability.
    - Supported by the maintainers of the original Win32 program. See
      https://pwsafe.org/

$ dpkg -I passwordsafe-ubuntu-0.98.1BETA.amd64.deb
  new debian package, version 2.0.
[...]
  Description: Password Safe is a password database utility.

$ dpkg -I passwordsafe-ubuntu-0.97BETA.amd64.deb > 97.txt
$ dpkg -I passwordsafe-ubuntu-0.98.1BETA.amd64.deb > 98.txt
$ diff -u 9{7,8}.txt
--- 97.txt	2016-04-02 12:23:02.526515044 -0400
+++ 98.txt	2016-04-02 12:23:07.558300712 -0400
@@ -1,24 +1,16 @@
   new debian package, version 2.0.
- size 3180436 bytes: control archive=743 bytes.
-     750 bytes,    19 lines      control
-     232 bytes,     6 lines   *  postinst             #!/bin/bash
+ size 11153514 bytes: control archive=1336 bytes.
+     410 bytes,    11 lines      control
+    2041 bytes,    28 lines      md5sums
       158 bytes,     5 lines   *  prerm                #!/bin/bash
- Package: passwordsafe
- Version: 0.97BETA
+ Package: pwsafe
+ Version: 0.98.1-BETA
   Section: utils
   Priority: optional
- Conflicts: pwsafe
   Architecture: amd64
- Installed-Size: 5000
- Maintainer: Rony Shapiro <ronys@users.sf.net>
+ Depends: libc6 (>= 2.7-18lenny4), libuuid1 (>= 1.41.3-1), 
libwxgtk3.0-0 (>= 3.0.0-2), libxtst6 (>= 2:1.0.3-1), libxerces-c3.1 (>= 
3.1.1-1+b1), libykpers-1-1 (>= 1.3.2-1squeeze1)
   Homepage: https://pwsafe.org/
- Depends: libc6 (>= 2.11.1-0ubuntu7.2), libuuid1 (>= 2.17.2-0ubuntu1), 
libwxgtk3.0-0 (>= 3.0.0-2), libxtst6 (>= 2:1.1.0-2), libxerces-c3.1 (>= 
3.1.0-1), libykpers-1-1 (>= 1.7.0-1)
- Description: Portable version of popular password manager program
-  pwsafe is the Linux version of the popular Windows PasswordSafe
-  password manager.
-  .
-  Features:
-   - Compatible with 1.x, 2.x and 3.x versions of the database format.
-   - Uses wxWidgets for portability.
-   - Supported by the maintainers of the original Win32 program. See
-     https://pwsafe.org/
+ Installed-Size: 13919
+ Maintainer: ronys@pwsafe.org
+ Description: Password Safe is a password database utility.
+

$ mkdir 97 98
$ dpkg -x ../passwordsafe-ubuntu-0.97BETA.amd64.deb 97
$ dpkg -x ../passwordsafe-ubuntu-0.98.1BETA.amd64.deb 98
$ diff -qr 97 98
Files 97/usr/bin/pwsafe and 98/usr/bin/pwsafe differ
Only in 97/usr/share/doc/passwordsafe: README.txt
Only in 97/usr/share/doc/passwordsafe: ReleaseNotes.txt
Only in 98/usr/share/doc/passwordsafe: changelog.Debian
Only in 97/usr/share/doc/passwordsafe: changelog.Debian.gz
Only in 97/usr/share/doc/passwordsafe: changelog.gz
Only in 97/usr/share/doc/passwordsafe: copyright
Only in 98/usr/share/doc/passwordsafe/help: helpDE.zip
Files 97/usr/share/doc/passwordsafe/help/helpEN.zip and 
98/usr/share/doc/passwordsafe/help/helpEN.zip differ
Only in 98/usr/share/doc/passwordsafe/help: helpES.zip
Only in 98/usr/share/doc/passwordsafe/help: helpFR.zip
Only in 98/usr/share/doc/passwordsafe/help: helpPL.zip
Only in 98/usr/share/doc/passwordsafe/help: helpRU.zip
Only in 98/usr/share/doc/passwordsafe/help: helpZH.zip
Files 97/usr/share/locale/da/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/da/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/de/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/de/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/es/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/es/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/fr/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/fr/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/it/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/it/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/ko/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/ko/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/nl/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/nl/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/pl/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/pl/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/ru/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/ru/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/sv/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/sv/LC_MESSAGES/pwsafe.mo differ
Files 97/usr/share/locale/zh/LC_MESSAGES/pwsafe.mo and 
98/usr/share/locale/zh/LC_MESSAGES/pwsafe.mo differ
Only in 98/usr/share/man/man1: pwsafe.1
Only in 97/usr/share/man/man1: pwsafe.1.gz
Files 97/usr/share/pwsafe/xml/KPV1_to_PWS.xslt and 
98/usr/share/pwsafe/xml/KPV1_to_PWS.xslt differ
Only in 98/usr/share/pwsafe/xml: KPV2_to_PWS.xslt
Files 97/usr/share/pwsafe/xml/pwsafe.xsl and 
98/usr/share/pwsafe/xml/pwsafe.xsl differ
Files 97/usr/share/pwsafe/xml/pwsafe_filter.xsd and 
98/usr/share/pwsafe/xml/pwsafe_filter.xsd differ
```

**Comment by ronys** [comment]
Ugh, my bad. Fixed in commit e197863.
